client/components/map.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+export default class Map extends React.Component {
```

Comment:
There is already a `Map` data structure defined in the global scope. Rename this to `RouteMap`.